lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +34,42 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
+    """
+    Create a verification attempt.
+
+    Args:
+        user (User): the user (usually a learner) performing the verfication attempt
+        name (string): the name of the user
+        status (string): the initial status of the verification attempt
+        expiration_datetime (datetime, optional): When the verification attempt expires. Defaults to None.
+
+    Returns:
+        VerificationAttempt (VerificationAttempt): The created VerificationAttempt instance
+    """
+    verification_attempt = VerificationAttempt.objects.create(
+        user=user,
+        name=name,
+        status=status,
+        expiration_datetime=expiration_datetime,
+    )
+
+    return verification_attempt.id
+
+
+def update_verification_status(attempt_id, status):
+    """
+    Update the VerificationAttempt status.
+
+    Arguments:
+        * id (str): the verification attempt id
+        * status (str): the new status
```

Comment:
Does there need to be validation that the status is correct here?

Reply:
That makes sense to me Isaac! Wasn't sure if this needed to happen here, especially if it's already happening elsewhere. :)

Reply:
I think that we should check that `status` is a valid status in both the `create` and `update` methods (i.e. it is one of `created`, `pending`, `approved`, or `denied`).

In terms of status transitions, I agree that it would be tough to assert a particular set of valid state transitions here, because each form of IDV will be different.